integration/spark/src/main/scala/org/apache/carbondata/spark/util/GlobalDictionaryUtil.scala

```diff
@@ -689,6 +689,10 @@ object GlobalDictionaryUtil extends Logging {
           generatePredefinedColDictionary(colDictFilePath, table,
             dimensions, carbonLoadModel, sqlContext, hdfsLocation, dictfolderPath)
         }
+        if (headers.length > df.columns.length) {
+          logError("Maybe either delimiter or fileheader is wrong")
+          throw new DataLoadingException("Maybe either delimiter or fileheader is wrong")
```

Comment:
@QiangCai ...Please modify this message as below:
"Either delimiter or fileheader provided is not correct"